core/templates/xregistry_openapi_template.json

```diff
@@ -1,1068 +1,1937 @@
+                {
+                    "in": "query",
+                    "name": "meta",
+                    "description": "If true, operate on the resource metadata sub-object (see spec: Resource Metadata vs Resource Document).",
+                    "required": false,
+                    "schema": { "type": "boolean" }
+                },
+                {
+                    "in": "query",
+                    "name": "noepoch",
+                    "description": "If true, epoch validation is ignored. (See spec: Creating or Updating Entities)",
+                    "required": false,
+                    "schema": { "type": "boolean" }
+                },
+                {
+                    "in": "query",
+                    "name": "nodefaultversionid",
+                    "description": "If true, any provided defaultversionid is ignored. (See spec: Default Version of a Resource)",
+                    "required": false,
+                    "schema": { "type": "boolean" }
+                },
+                {
+                    "in": "query",
+                    "name": "nodefaultversionsticky",
+                    "description": "If true, any provided defaultversionsticky value is ignored. (See spec: Default Version of a Resource)",
+                    "required": false,
+                    "schema": { "type": "boolean" }
+                },
+                {
+                    "in": "query",
+                    "name": "noreadonly",
+                    "description": "If true, attempts to update readonly attributes are silently ignored. (See spec: Resource Attributes)",
+                    "required": false,
+                    "schema": { "type": "boolean" }
+                },
+                {
+                    "in": "query",
+                    "name": "setdefaultversionid",
+                    "description": "Specifies the versionid to be set as the default version. (See spec: Default Version of a Resource)",
+                    "required": false,
+                    "schema": { "type": "string" }
+                },
+                {
+                    "$ref": "#/components/parameters/inline"
+                },
+                {
+                    "$ref": "#/components/parameters/filter"
+                },
+                {
+                    "$ref": "#/components/parameters/doc"
+                }
+            ],
+            "get": {
+                "operationId": "get{%-resourceNameSingular-%}",
+                "tags": ["Resource"],
+                "description": "Gets the resource (default version). Supports $details view via the 'doc' query parameter. (See spec: Retrieving a Resource and Retrieving a Resource as Metadata)",
+                "responses": {
+                    "200": {
+                        "description": "The resource (or a redirect if a RESOURCEurl is set).",
+                        "headers": {
+                            "resource-id": {
```

Comment:
Should this be "xRegistry-{%-resourceNameSingular-%}" ?
Same for the ones below. xRegistry-xxx

Reply:
How do we deal with hasdoc=false cases?